.github/workflows/component-build-images.yml

```diff
@@ -59,8 +59,8 @@ jobs:
             tag_suffix: adservice
             context: ./
             setup-qemu: true
-          - file: ./src/cartservice/src/Dockerfile
-            tag_suffix: cartservice
+          - file: ./src/cart/src/Dockerfile
+            tag_suffix: cart
```

Comment:
Is the plan to continue to support the Helm chart?

If so, I suggest leaving the tag_suffix as `cartservice` for now. There is a contract between the suffix this repository uses and the Helm chart.

You can change the name of the services everywhere else in the code leaving changing the `tag_suffix` for all the services for a single PR and then following up with a single fix to the Helm chart.

See #1848 where I propose reverting the tag suffix for the accounting service for now.

Reply:
The Helm chart is fixed on the last good release which is 1.12.0. We need the tag suffix changes here for local development to continue as we work through changing all these names.